granulate_utils/linux/cgroups/memory_cgroup.py

```diff
@@ -0,0 +1,28 @@
+#
+# Copyright (c) Granulate. All rights reserved.
+# Licensed under the AGPL3 License. See LICENSE.md in the project root for license information.
+#
+
+from granulate_utils.linux.cgroups.base_cgroup import BaseCgroup
+from granulate_utils.linux.cgroups.exceptions import MissingController
+
+
+class MemoryCgroup(BaseCgroup):
+    controller = "memory"
+
+    def get_memory_limit(self) -> int:
+        return int(self.read_from_controller("memory.limit_in_bytes"))
+
+    def get_max_usage_in_bytes(self) -> int:
+        return int(self.read_from_controller("memory.max_usage_in_bytes"))
+
+    def set_limit_in_bytes(self, limit: int) -> None:
+        self.write_to_controller("memory.limit_in_bytes", str(limit))
+        try:
+            self.write_to_controller("memory.memsw.limit_in_bytes", str(limit))
+        except MissingController:
```

Comment:
Exception is confusing here - it's not that the memory controller is missing, but the specific file is missing. We can split it to 2 exceptions - one is `MissingSubsystem` for when you try to use `cpu,cpuacct` e.g and that's non existent, and one when you try to use a file in a subsystem and the file itself isn't present - that's another error that needs to be distinguished.